app/views/layouts/application.slim

```diff
@@ -33,7 +33,7 @@ html
           li : a href=static_path("about") Про нас
           li : a href=static_path("helping-us") Як можна допомогти
 
-        div © 2017 Українські книжки
+        div © 2018 Українські книжки
```

Comment:
Shouldn't this be 2017–2018? 
I know it doesn't make [any difference in legal sense](https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated), but at least foundation year would be visible.

Reply:
Sure 👍